resources/migrations/001_update_migrations.yaml

```diff
@@ -7948,7 +7948,18 @@ databaseChangeLog:
 #
 # ADVICE:
 #
-# 1) Run ./bin/lint-migrations-file.sh to run core.spec checks against any changes you make here. Liquibase is pretty
+# 1) Think through some of these points when writing a migration, learn from our past mistakes:
+#    - If you need data migration, do you really need to? Could the feature work without it? Prefer not to if possible.
+#      Data in the wild can be vastly different from what you expect, and it's hard to get right.
+#    - If you still need to, make sure it works when Metabase is encrypted. Fields like `metabase_database.details`
+#      or `setting.value` can be encrypted, so you need to decrypt them in your migration. See #42617, #44048.
+#    - Prefer SQL migrations over custom migrations.
+#    - Postgres and MySQL have their differences. Make sure your migration works on both.
```

Comment:
+ h2?

Reply:
(usually more similar to postgres than mysql but I think I've hit differences before)

Reply:
added h2